java/util/pom.xml

```diff
@@ -4,110 +4,72 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.google</groupId>
-    <artifactId>google</artifactId>
-    <version>1</version>
+    <groupId>com.google.protobuf</groupId>
+    <artifactId>protobuf-parent</artifactId>
+    <version>3.0.0-beta-2</version>
   </parent>
-  <groupId>com.google.protobuf</groupId>
+
   <artifactId>protobuf-java-util</artifactId>
-  <version>3.0.0-beta-2</version>
   <packaging>bundle</packaging>
-  <name>Protocol Buffer Java API</name>
-  <description>
-    Protocol Buffers are a way of encoding structured data in an efficient yet
-    extensible format.
-  </description>
-  <inceptionYear>2008</inceptionYear>
-  <url>https://developers.google.com/protocol-buffers/</url>
-  <licenses>
-    <license>
-      <name>New BSD license</name>
-      <url>http://www.opensource.org/licenses/bsd-license.php</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
-  <scm>
-    <url>https://github.com/google/protobuf</url>
-    <connection>
-      scm:git:https://github.com/google/protobuf.git
-    </connection>
-  </scm>
+
+  <name>Protocol Buffers [Util]</name>
+  <description>Utilities for Protocol Buffers</description>
+
   <dependencies>
     <dependency>
-      <groupId>com.google.protobuf</groupId>
+      <groupId>${project.groupId}</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.0.0-beta-2</version>
-      <scope>compile</scope>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>protobuf-java</artifactId>
```

Comment:
We're not ... the first one is the main artifact, the second one is the test-jar artifact (see the type)